src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,67 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (newPage: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const linksAmount = Math.ceil(total / perPage);
+  const linkNumbers = getNumbers(1, linksAmount);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
```

Comment:
Consider using a button instead of an anchor for the pagination controls. Buttons are more appropriate for actions that do not navigate to a different URL.